src/ripple/app/main/Application.cpp

```diff
@@ -456,10 +456,11 @@ class ApplicationImp
             ( *this, stopwatch()
             , *m_jobQueue
             , m_collectorManager->collector ()
-            , [this](uint256 const& setHash,
-                std::shared_ptr <SHAMap> const& set)
+            , [this](std::shared_ptr <SHAMap> const& set,
```

Comment:
Its kind of annoying that a piece of business logic is wedged in Application's ctor-initializer list.